pyproject.toml

```diff
@@ -31,13 +31,13 @@ classifiers = [
 keywords = ["IDAES", "energy systems", "chemical engineering", "process modeling"]
 dynamic = ["version"]
 dependencies = [
-    "idaes-pse >= 2.3.0rc1",
+    "idaes-pse >= 2.4.0",
```

Comment:
Just to check that these are expected for this PR - these look like they really belong in a separate PR to update versions.